src/hooks/useScheduleData.js

```diff
@@ -1,13 +1,16 @@
 import { useState, useEffect, useCallback } from "react";
+import moment from "moment";
 
 export const useScheduleData = () => {
   const [airTimeSchedule, setAirtimeSchedule] = useState(null);
+  const [fiveDayAirTimeSchedule, setFiveDayAirtimeSchedule] = useState(null);
   const [scheduleData, setScheduleData] = useState(null);
 
   const scheduleApiCall = useCallback(() => {
     fetch("https://ehfm.airtime.pro/api/week-info")
       .then((response) => response.json())
       .then((data) => {
+        delete data.AIRTIME_API_VERSION;
```

Comment:
What's this about?

Reply:
The response returns API version, but later I'm looping through the `Object.keys` and I'm only interested in the days keys. 

<img width="462" alt="Screenshot 2022-11-13 at 20 57 03" src="https://user-images.githubusercontent.com/22624697/201544204-06f6b892-b427-4969-ae38-70b18c613d2d.png">